.circleci/config.base.yml

```diff
@@ -1325,6 +1345,27 @@ workflows:
                 - /run-e2e-with-rc\/.*/
                 - /tagged-release\/.*/
                 - /tagged-release-without-e2e-tests\/.*/
+      - deployment_verification_release:
+          requires:
+            - publish_to_npm
+            - github_release
+            - upload_pkg_binaries
+          filters:
+            branches:
+              only:
+                - release
+      - deployment_verification_prerelease:
+          requires:
+            - publish_to_npm
+            - github_release
+            - upload_pkg_binaries
+          filters:
+            branches:
+              only:
+                - /release_rc\/.*/
+                - /run-e2e-with-rc\/.*/
+                - /tagged-release\/.*/
+                - /tagged-release-without-e2e-tests\/.*/
```

Comment:
We don't publish to pre-release on github for tagged releases.

Reply:
I clarified the naming here. Tagged and RC releases do not get deployed to github. They need to be treated differently so the verification doesn't fail when checking github.